Modules/EMCAL/src/ClusterTask.cxx

```diff
@@ -39,8 +43,11 @@
 
 #include <DataFormatsEMCAL/AnalysisCluster.h>
 
+#include <EMCALReconstruction/Clusterizer.h> //svk
+
 std::unique_ptr<o2::emcal::EventHandler<o2::emcal::Cell>> mEventHandler;
 std::unique_ptr<o2::emcal::ClusterFactory<o2::emcal::Cell>> mClusterFactory;
+std::unique_ptr<o2::emcal::Clusterizer<o2::emcal::Cell>> mClusterizer; //svk
```

Comment:
You must not overwrite the members declaring global variables. Despite that this shadows the class members global variables are among the most dangerous you can do. Never use global variables except from simple macros.

Reply:
Yes, I agree with this comment and I have removed the lines declaring the members as global variables.